### PR TITLE
add LSL changes from LeTigre 15.09.14.305056

### PR DESCRIPTION
**Pull request by buildersbrewery**
http://wiki.secondlife.com/wiki/Release_Notes/Second_Life_RC_LeTigre/15#15.09.14.305056

### CODE REVIEW

**Comment by buildersbrewery** [comment]
see commit messages for details

File: lib/ace/mode/lsl_highlight_rules.js [hunk 1]
```diff
@@ -111,7 +111,8 @@ function LSLHighlightRules() {
                 regex : "\\*\\/",
                 next : "start"
             }, {
-                defaultToken : "comment.block.lsl"
+                token : "comment.block.lsl",
+                regex : ".+"
```
Comment (nightwing) [inline lib/ace/mode/lsl_highlight_rules.js]: Why is this needed?

Comment (buildersbrewery) [inline lib/ace/mode/lsl_highlight_rules.js]: It had appeared to be clearer definition for me.

Comment (nightwing) [inline lib/ace/mode/lsl_highlight_rules.js]: It doesn't work, try `/* comment */ still comment?`

Comment (buildersbrewery) [inline lib/ace/mode/lsl_highlight_rules.js]: sorry about that, pushed update...
